[IMP] referrals: update pics and remove save button

larm-odoo opened the pull request:
The Referrals app no longer uses Save buttons, and the Sequence field in the avatar forms have been removed. Images have been pudated.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/8686).

larm-odoo commented:
@brse-odoo this is ready for a first look! I updated most of the images, removed a reference to a sequence field that is no longer in V17, and I removed references to Save and Edit buttons that are no longer required.

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
-indicate the order in which the slides should be displayed. A :guilabel:`Company` may be selected,
-however, if this field is populated, that slide will only be displayed for that particular company.
+Make any changes to the message in the :guilabel:`Text` field. A :guilabel:`Company` may be
+selected, however, if this field is populated, that slide will only be displayed for that particular
```
```suggestion
selected, however, if this field is populated, that slide will *only* be displayed for that particular
```
Optional for emphasis

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
 
-The image can be modified as well. Hover over the image thumbnail in the top right corner of the
+.. note::
+   The :guilabel:`Company` field only appears when in a multi-company database.
```
```suggestion
   The :guilabel:`Company` field **only** appears when in a multi-company database.
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
+To cancel any changes made, click the :guilabel:`↺ (Discard changes)` icon to delete any changes and
+revert to the original content.
+
+The sequence the slides appear can be changed from the Onboarding dashboard. Click the up and down
```
```suggestion
The sequence in which the slides appear can be changed from the Onboarding dashboard. Click the up and down
```
Suggestion:
If we're on the onboarding dashboard: change to :guilabel:`Onboarding dashboard`
or, if we're not on the dashboard,
change to: *Onboarding* dashboard

larm-odoo replied:
Good suggestion- we WERE but then I had people jump to the actual individual slide form. So italics itis!

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -103,7 +105,7 @@ Share job positions
 
 In order to earn referral points, a user must first share a job position so the applicant can apply
-for the position. There are several ways to share job positions within the Referrals application,
+for the position. Job positions can be shared in several ways within the Referrals application,
```
```suggestion
for the position. Job positions can be shared in several ways within the *Referrals* application,
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -166,8 +168,8 @@ pop-up window.
 
-There are several other ways to share a job position aside from sending an email, via social media
-platforms and tracking links to the job position. At the bottom of each job position card are four
+Other than sending an email, job positions can be shared in several other ways, via social media
+platforms, and tracking links to the job position. At the bottom of each job position card are four
```
```suggestion
Other than sending an email, job positions can be shared via social media
platforms, and tracking links to the job position. At the bottom of each job position card are four
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
-and lastly, if an activity type was selected, select the specific activity.
+Type in any additional information to add to the post, in the space directly below the user's name.
+If desired, :guilabel:`tag` users, :guilabel:`check-in` at a location, or add a :guilabel:`status`
+to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag to
+the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
```
```suggestion
to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag in
the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
+to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag to
+the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
+from the pop-up of predetermined locations based on the user's current location. To add a
+:guilabel:`status` click the :guilabel:`Feeling/Activity` icon, then select either the feeling or
```
```suggestion
:guilabel:`status`, click the :guilabel:`Feeling/Activity` icon, and select either the feeling or
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
+the activity type, and lastly, if an activity type was selected, select the specific activity.
 
-There are five methods to share the job position, :guilabel:`Your Story`, :guilabel:`Share on a
+Five methods are available to share the job position, :guilabel:`Your Story`, :guilabel:`Share on a
```
```suggestion
Five methods are available to share the job position: :guilabel:`Your Story`, :guilabel:`Share on a
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -219,13 +221,13 @@ located at the top of the page, directly beneath the :guilabel:`Share on Faceboo
-      buttons are greyed out. Activate the radio button(s) to indicate how the job position should
-      be shared, either in the news feed, the story, or both.
+      buttons are grayed out. Activate the radio buttons to indicate how the job position should be
+      shared, either in the news feed, the story, or both.
```
```suggestion
      shared: either in the news feed, the story, or both.
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -303,21 +305,21 @@ The default message is:
-There are various other items that can be added to the message. Media (photos, videos, etc), GIF's,
-polls, emojis, and a location may all be added. Click the corresponding icon in the bottom left of
-the message and follow the prompts to add the desired additional elements.
+Various other items that can be added to the message. Media (photos, videos, etc), GIFs, polls,
```
```suggestion
Various other items that can be added to the message. Media (photos, videos, etc.), GIFs, polls,
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -384,14 +386,17 @@ Send as private message
+number of connections that can be messaged.
+
+Next, enter any additional text in the message body. Click :guilabel:`Send` in the lower-right
+corner to send the message. A :guilabel:`Message successfully sent!` screen loads. Two options are
```
```suggestion
corner to send the message. Once sent, a :guilabel:`Message successfully sent!` screen loads. Two options are
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -403,22 +408,22 @@ instead of one job position at a time. To do this, navigate to the main *Referra
-will be sent and the window will close.
+The `(Company)/Job Offers` text is a tracking link to a complete list of all job positions currently
+being recruited for, that are live on the company's website (published). Add any additional text
+and make any edits to the message body, then click :guilabel:`Send Mail` to send the email. The
```
```suggestion
and make any edits to the message body, if necessary. Then, click :guilabel:`Send Mail` to send the email. This
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -403,22 +408,22 @@ instead of one job position at a time. To do this, navigate to the main *Referra
+The `(Company)/Job Offers` text is a tracking link to a complete list of all job positions currently
+being recruited for, that are live on the company's website (published). Add any additional text
+and make any edits to the message body, then click :guilabel:`Send Mail` to send the email. The
+message will be sent and the window will close.
```
```suggestion
sends the message and closes the window.
```

brse-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -730,7 +746,7 @@ Click :guilabel:`Create` and a blank alert form loads. Enter the following infor
   remains blank, the alert is visible to everyone with access to the Referrals application. If a
   company is specified, only user's within that company (who also have access to the Referrals
-  application) will see the alert.
+  application) will see the alert. This field only appears when in a multi-company environment.
```
```suggestion
  application) see the alert. This field **only** appears when in a multi-company database.
```
Changed environment to database to keep consistent with how it was written earlier - but choose which one you like better :-)

brse-odoo reviewed (approved):
Hi @larm-odoo, I've finished my review of the specific changes you made to this doc - great job! I'm approving with small suggestions for you to accept/reject as you see fit. The new screenshots all look great too. Let me know if you have any questions, thanks!

larm-odoo commented:
After first round review and the robots, this is ready for a final review, @ksc-odoo!

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -4,17 +4,17 @@ Referrals
 Any user with either :guilabel:`Referral User`, :guilabel:`Officer`, or :guilabel:`Administrator`
-rights for the Recruitment application has access to the Referrals application. Only user's with
+rights for the Recruitment application has access to the Referrals application. Only users with
 :guilabel:`Administrator` rights for the Recruitment application have access to the :ref:`reporting
```
```suggestion
:guilabel:`Administrator` access rights for the *Recruitment* application have access to the :ref:`reporting
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
 the Recruitment application can modify onboarding slides. To edit a slide, navigate to
```
```suggestion
the *Recruitment* application can modify onboarding slides. To edit a slide, navigate to
```

ksc-odoo replied:
Not sure if the sentence: "To edit a slide, navigate to the Recruitment application can modify onboarding slides."
Can you please rework this?

larm-odoo replied:
This is the full text in this section:

To edit a slide, navigate to :menuselection:`Referrals app --> Configuration --> Onboarding.` Each line displays the text for the individual onboarding slide. To edit an onboarding slide, click on an individual slide line to open the slide's onboarding form.


I don't see what is missing?

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
-indicate the order in which the slides should be displayed. A :guilabel:`Company` may be selected,
-however, if this field is populated, that slide will only be displayed for that particular company.
+Make any changes to the message in the :guilabel:`Text` field. A :guilabel:`Company` may be
+selected, however, if this field is populated, that slide will *only* be displayed for that
```
```suggestion
selected, however, if this field is populated, that slide is *only* displayed for that
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
+.. note::
+   The :guilabel:`Company` field only appears when in a multi-company database.
+
+The image can be modified as well. Hover over the image thumbnail in the top-right corner of the
```
```suggestion
The image can be modified, as well. Hover over the image thumbnail in the top-right corner of the
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
-checkbox and before the slide text, and drag the slide to the desired position. The
-:guilabel:`Sequence` automatically changes on all the slide's onboarding forms, reflecting the new
-sequence.
+The slide can be saved manually at any time by clicking the *save manually* option, represented by a
+:guilabel:`(cloud upload)` icon, located at the end of the slide description in the top-left corner.
+To cancel any changes made, click the :guilabel:`↺ (Discard changes)` icon to delete any changes and
+revert to the original content.
```
I don't think this section is necessary, as this function appears on nearly each page in the database. So, I think you can delete this.

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
+To cancel any changes made, click the :guilabel:`↺ (Discard changes)` icon to delete any changes and
+revert to the original content.
+
+The sequence in which the slides appear can be changed from the *Onboarding* dashboard. Click the up
+and down arrow symbol that appears after the checkbox and before the slide text, and drag the slide
+to the desired position.
```
I'm not seeing an up/down arrow -- just the typical "six dot" icon that needs to be clicked to activate the drag-and-drop...can you please double-check this during your revisions?

larm-odoo replied:
Good catch! I originally was only updating images, then added some more changes, so I missed this!

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -103,7 +105,7 @@ Share job positions
 In order to earn referral points, a user must first share a job position so the applicant can apply
-for the position. There are several ways to share job positions within the Referrals application,
+for the position. Job positions can be shared in several ways within the *Referrals* application,
 through the :ref:`View Jobs <referrals/view-jobs>` button and the :ref:`Email A Friend
 <referrals/email-jobs>` button.
```
where in the application are these buttons found? a little specificity here would be helpful

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -166,10 +168,10 @@ pop-up window.
-platforms and tracking links to the job position. At the bottom of each job position card are four
-icons and corresponding tracking links that can be used to share the job position, keeping track of
-applicants in the Referrals application.
+Other than sending an email, job positions can be shared via social media platforms, and tracking
```
```suggestion
Other than sending an email, job positions can be shared, via social media platforms, and tracking
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -166,10 +168,10 @@ pop-up window.
-icons and corresponding tracking links that can be used to share the job position, keeping track of
-applicants in the Referrals application.
+Other than sending an email, job positions can be shared via social media platforms, and tracking
+links to the job position. At the bottom of each job position card are four icons and corresponding
+tracking links that can be used to share the job position, keeping track of applicants in the
+*Referrals* application.
```
```suggestion
links to the job position. At the bottom of each job position card are four icons, and corresponding
tracking links, that can be used to share the job position, keeping track of applicants in the
*Referrals* application.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
 with the link populated in the main body of the new post. If the user is *not* already logged in, a
 log-in screen loads instead, prompting the user to log-in to Facebook first.
```
```suggestion
log-in screen loads, instead, prompting the user to log-in to Facebook first.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
-the pop-up of predetermined locations based on your current location. To add a :guilabel:`status`,
-click the :guilabel:`Feeling/Activity` icon, then select either the feeling or the activity type,
-and lastly, if an activity type was selected, select the specific activity.
+Type in any additional information to add to the post, in the space directly below the user's name.
```
```suggestion
Type in any additional information to add to the post in the space directly below the user's name.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
-click the :guilabel:`Feeling/Activity` icon, then select either the feeling or the activity type,
-and lastly, if an activity type was selected, select the specific activity.
+Type in any additional information to add to the post, in the space directly below the user's name.
+If desired, :guilabel:`tag` users, :guilabel:`check-in` at a location, or add a :guilabel:`status`
+to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag in
+the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
+from the pop-up of predetermined locations based on the user's current location. To add a
+:guilabel:`status`, click the :guilabel:`Feeling/Activity` icon, and select either the feeling or
+the activity type, and lastly, if an activity type was selected, select the specific activity.
```
I don't believe 'tag', 'check-in', nor 'status' need a GUI, since that is not how those fields appear on the UI

Can you also break up this large paragraph, please?

Describe the check in icon

mention "window" (or "form" when applicable) after every mention of "pop-up"

can you describe the look of the 'feeling/activity' icon?

ksc-odoo reviewed (changes requested):
Hey there, @larm-odoo -- pausing my Final Review here for a couple reasons...

- I think some added specificity is needed in the indicated areas mentioned in the feedback

- While I fully/always appreciate your relentless attention to detail, when you go into the minute details about posting for the various social media platforms, that's where things may be getting "off topic" -- they are reading more like Facebook/X/etc. docs, and teaching the user how to post, when I think a simple mention of the button/share capabilities would suffice. That would also greatly cut down the size of this rather large doc, making things more concise and easier for the reader to digest.

Once you cut all those elements down, and make the other suggested edits left in the first portion of the doc, tag me again, and I'll finish the Final Review.

Thanks! 👍

larm-odoo commented:
Hi @ksc-odoo - thanks for the review! This is ready for another look. I implemented all the requested changes, except for the part where you asked for changes in the social media sharing sections, which yu suggested were trimmed down. I removed a lot of those sections, so they are more "odoo" focused and it says to post accoridng to those other apps/sites, etc. I think it's more concise this way as well.

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -516,12 +355,18 @@ To see all the referrals (both ongoing and successful) click the :guilabel:`Refe
 
-Each referral card lists the name of the applicant, the department the job position is in, the name
-of the job position, the person responsible for filling the role (recruiter), and the points earned.
+Each referral card lists the name of the applicant, the subject/application title as it appears on
+the applicant card in the *Recruitment* app, the name of the job position, the person responsible
+for filling the role (recruiter), and the points earned. For referrals that have been hired, the
+selected avatar also appears on the card.
```
```suggestion
Each referral card lists the name of the applicant, the subject/application title (as it appears on
the applicant card in the *Recruitment* app), the name of the job position, the person responsible
for filling the role (recruiter), and the points earned. For referrals that have been hired, the
selected avatar also appears on the card.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -548,7 +393,7 @@ check mark appears next to the stage.
 
    The total points earned for a referral that is hired is 85 points. Modifications of the points
-   earned for each stage is done in the Recruitment application. Refer to the
+   earned for each stage are done in the Recruitment application. Refer to the
```
```suggestion
   earned for each stage are done in the *Recruitment* application. Refer to the
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -562,13 +407,13 @@ Create rewards
 The rewards are the only configurations needed when setting up the Referrals application. Only users
```
```suggestion
The rewards are the only configurations needed when setting up the *Referrals* application. Only users
```
again, please go through this PR to ensure all application titles are formatted correctly. I will stop doing them personally from here on out --  but please double-check their accuracy during your revisions. thanks!

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -562,13 +407,13 @@ Create rewards
 with :guilabel:`Administrator` rights for the Recruitment application can create or modify rewards.
 To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
-:guilabel:`Create` and a reward form loads. Enter the following information on the form:
+:guilabel:`New` and a reward form loads. Enter the following information on the form:
```
```suggestion
:guilabel:`New`, and a reward form loads. Enter the following information on the form:
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -562,13 +407,13 @@ Create rewards
-:guilabel:`Create` and a reward form loads. Enter the following information on the form:
+:guilabel:`New` and a reward form loads. Enter the following information on the form:
 
 - :guilabel:`Product Name`: Enter the name as it should appear for the reward.
```
please make sure that all words that immediately follow a ":" are lower-case.
```suggestion
- :guilabel:`Product Name`: enter the name as it should appear for the reward.
```
just fixing this one -- ensure similar formatting follows suit throughout the PR. thanks!

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -562,13 +407,13 @@ Create rewards
 
 - :guilabel:`Product Name`: Enter the name as it should appear for the reward.
 - :guilabel:`Cost`: Enter the amount of points required to redeem the reward.
 - :guilabel:`Company`: Using the drop-down menu, select the company the reward will appear for. If a
```
```suggestion
- :guilabel:`Company`: Using the drop-down menu, select the company the reward appears for. If a
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -562,13 +407,13 @@ Create rewards
+  with the company listed on the form. This field only appears if in a multi-company environment.
 - :guilabel:`Gift Responsible`: Select the person who will be responsible for procuring and
   delivering the reward to the recipient, using the drop-down menu. This person is alerted when the
   reward is purchased, so they know when to deliver the reward to the recipient.
```
just to clarify...are these rewards "purchased" or "earned" via points received?

larm-odoo replied:
ah, I see - it's earned using the rewards- you have to "spend" the points you earned.

larm-odoo replied:
I changed it to "boughtin the *Referrals* app" sinc ethe word in the app is "buy" so the nomenclature matches but it is specific to being "bought" IN the app- so I think that clarifies it.

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -579,7 +424,7 @@ To add rewards, navigate to :menuselection:`Referrals app --> Configuration -->
   Click the :guilabel:`✏️ (pencil)` icon and a file explorer appears. Navigate to the photo file,
```
```suggestion
  Click the :guilabel:`✏️ (pencil)` icon, and a file explorer appears. Navigate to the photo file,
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -643,7 +489,7 @@ Odoo superheroes, and each level adds an additional element to their avatar, suc
-in the list to open the level form. Click :guilabel:`Edit` to make modifications.
+in the list to open the level form, then make modifications.
 
 Type in the name or number of the level in the :guilabel:`Level Name` field. What is entered is
```
```suggestion
Type in the name (or number) of the level in the :guilabel:`Level Name` field. What is entered is
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -656,8 +502,10 @@ If desired, the :guilabel:`Image` can also be modified. Hover over the image to
 (pencil)` icon and a file explorer window appears. Navigate to the desired image file then click
 :guilabel:`Open` to select it.
```
```suggestion
(pencil)` icon, and a file explorer window appears. Navigate to the desired image file, then click
:guilabel:`Open` to select it.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -471,8 +308,10 @@ modified. Hover over the image being replaced to reveal a :guilabel:`✏️ (pen
-:guilabel:`Discard` button to revert to the original information.
+The referral friend form automatically saves but can be saved manually at any time by clicking the
+*save manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left
+corner. To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete
```
is it an 'X' or a 'counter-clockwise arrow'?

larm-odoo replied:
Still an X- I checked in the latest (Runbot 17.2)

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -656,8 +502,10 @@ If desired, the :guilabel:`Image` can also be modified. Hover over the image to
-:guilabel:`Discard` button to revert to the original information.
+The level form saves automatically but can be saved manually at any time by clicking the *save
+manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left corner.
+To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete any
```
is it an 'X' or a 'counter-clockwise arrow'?

larm-odoo replied:
Yup- it's an X

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -656,8 +502,10 @@ If desired, the :guilabel:`Image` can also be modified. Hover over the image to
+The level form saves automatically but can be saved manually at any time by clicking the *save
+manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left corner.
+To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete any
+changes and revert to the original content.
```
```suggestion
changes, and revert to the original content.
```

ksc-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -656,8 +502,10 @@ If desired, the :guilabel:`Image` can also be modified. Hover over the image to
 
-Once all changes have been made, click the :guilabel:`Save` button to save the changes, or click the
-:guilabel:`Discard` button to revert to the original information.
+The level form saves automatically but can be saved manually at any time by clicking the *save
```
```suggestion
The level form saves automatically, but can be saved manually at any time by clicking the *Save
```

ksc-odoo reviewed (approved):
hey there, @larm-odoo -- just finished giving this another look.  I am approving now, but there a decent amount of minor changes and adjustments that require your attention. There is also a few formatting issues that should be double-checked and changed throughout the doc, as well. But, that said, I trust your diligent attention to detail, and don't think you'll have any issues making the necessary adjustments. 👍 Once you think it's fully-fixed and ready to go, feel free to tag this for Tech Review. Thanks!

larm-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -181,217 +179,56 @@ Link
 X/Twitter
 ~~~~~~~~~
 
 A job position can also be shared on X/Twitter. Click the :guilabel:`Share Now` button with the bird
```
OMG! Iput a ticket in and it was going to be done at some point, but I was not aware it already happened! Thank you- I am glad they changed that (I'll update my image now, too!)

larm-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -403,22 +240,22 @@ instead of one job position at a time. To do this, navigate to the main *Referra
+`There are some amazing job offers in my company! Have a look, they can be interesting for you\:`
 
-`See Job Offers`
+`(Company)/Job Offers`
```
I thnk this was either because of multi-company, or just how it looked in previous versions. Updating!

larm-odoo commented on content/applications/hr/referrals.rst:
```diff
@@ -102,10 +100,10 @@ sequence.
+for the position. Job positions can be shared in several ways within the *Referrals* application,
 through the :ref:`View Jobs <referrals/view-jobs>` button and the :ref:`Email A Friend
-<referrals/email-jobs>` button.
+<referrals/email-jobs>` button, located at the bottom of the *Referrals* app dashboard.
```
Good idea- adding  note block

larm-odoo commented:
@samueljlieber this is ready for a tech review! FYI- I see the error in the build, BUT, it's because I have a colon inside a tck mark- but it is in the email text, so I'd like to keep it in.

samueljlieber reviewed (changes requested):
Hi @larm-odoo! Just a couple quick fixed and this PR will be all set. Nice job with these content improvements. Tag me for another look once these are addressed, thank you!

larm-odoo commented:
@samueljlieber - thanks for the review! Everything is addressed/updated. Ready for ya!

samueljlieber commented on content/applications/hr/referrals.rst:
```diff
@@ -181,217 +182,56 @@ Link
-      shared with groups the user has not joined. Begin to type in the name of the group. As the
-      group name is typed, a list appears, populated with the group names that match what is
-      entered. Select the group from the list. Only one group can be selected.
+:guilabel:`f (Facebook icon)` inside it.
```
```suggestion
:guilabel:`f (Facebook)` icon inside it.
```

samueljlieber reviewed (approved):
Hi @larm-odoo, looks like the two `:guilabel:` suggestions in my previous review were missed. Please address these before merging. Nice job resolving the build error and fixing the image! Approving and delegating to you 👍
..
@robodoo delegate=larm-odoo

larm-odoo commented:
@robodoo r+
